packages/asset-service/sample.env

```diff
@@ -1 +1,2 @@
-REACT_APP_ETHEREUM_NODE_URL=https://daemon.ethereum.shapeshift.com
+ETHEREUM_NODE_URL=https://daemon.ethereum.shapeshift.com
```

Comment:
Nothing react about this, this isn't injected and is local to asset-service as a separate package, renamed this one while at it